indico/util/string.py

```diff
@@ -32,6 +36,95 @@
 from sqlalchemy import ForeignKeyConstraint, inspect
 
 
+class AutoLinkExtension(markdown.extensions.Extension):
+    def __init__(self, rules: list[list], **kwargs):
+        self.rules = rules
+        super().__init__(**kwargs)
+
+    # flake8: noqa: N802
+    def extendMarkdown(self, md: markdown.Markdown):
```

Comment:
This syntax does not exist. So it's parsed as `flake8: noqa` which disabled flake8 (all rules!) for the whole file.

```suggestion
    def extendMarkdown(self, md: markdown.Markdown):  # noqa: N802
```